### PR TITLE
Shihao - Lint Fix Header

### PR DESCRIPTION
**Pull request by sxiong5**
# Description
Lint Fix `src/component/Header/**`

## Main changes explained:
- Change the code style according to ESLint and Prettier

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any user
4. go to dashboard
5. check the app works as normal (specific the `Header`)
6. set up ESLint and Prettier in VSCode
7. check all files in `src/component/Header/**` doesn't have errors

## Screenshots or videos of changes:
Screenshot 2023-08-30 at 15 06 17 (2)

### CODE REVIEW

**Comment by iTvX** [comment]
I have tested your pr and followed your instructions. I don't find any error.

CleanShot 2023-08-30 at 16 39 20@2x

**Review by DavidC0126 — COMMENTED** [review]
I tested it and it works as intended.
PR1202

**Comment by Jiyuanxia** [comment]
Works perfectly!
Screenshot 2023-08-30 211636

**Review by GaryB93 — APPROVED** [review]
Great work in cleaning up the codebase by destructuring props object! No lint errors found in Header component and functionality has not been affected. Header component removed from .eslintignore & .prettierignore files.

Screenshot 2023-08-31 at 2 56 27 PM

**Comment by gabi-lima** [comment]
Shouldn't the username appear in the header? Right after Welcome, 'user...'

**Comment by AnishPandita** [comment]
Hey, followed the instructions and didn't see any errors due to the PR. The code looks nice as well.
Screenshot 2023-08-31 at 4 51 20 PM

**Comment by sxiong5** [comment]
> Shouldn't the username appear in the header? Right after Welcome, 'user...'

Hey @gabi-lima, it should display the name like "Welcome, xxx". But this is a backend issue since the first name in the data is empty.

Screenshot 2023-09-01 at 00 20 02

**Review by ramyaram2092 — CHANGES_REQUESTED** [review]
Hello @shihao firstly good job 
The code looks clean. The header looks error free. 
However I do think the name is not getting populated near Welcome ..
I checked the development branch . I can see the name there 
So could you recheck the fix you did especially for this field

image

**Review by yaow62023 — COMMENTED** [review]
PR tested, No error show up. However, there is no first name after word "Welcome, ". Check the picture below for details.

**Review by luisarevalo21 — DISMISSED** [review]
Hello @sxiong5 ,

I tested your PR and works as intended. Great work. 
The application works as intended. 
As others mentioned the header is missing the name, but since you pointed out it is a backend issue, it is unrelated to this PR.

Great work. 
Screenshot 2023-09-01 at 8 39 40 PM

**Comment by Shreyj14** [comment]
Hi @sxiong5 , as mentioned by others above. Please check for the user name. Apart from that it looks good.

Screenshot 2023-09-02 at 2 13 10 PM

**Review by vikrambadhan — APPROVED** [review]
Hi, I reviewed the header code and tested your PR. Everything works perfectly!

Week2- PR2

**Review by Sssky-18 — DISMISSED** [review]
Hi @sxiong5, I checked this code and there was no errors. When running the dashboard, I can see the header correctly and my name was shown behind the "Welcome," so I think your recent fix worked as expected.

PR#1202

**Review by dah00 — DISMISSED** [review]
hey @sxiong5, I tested your PR, I don't see any errors and everything is displayed as expected. Also, my name appears after the Welcome. Good work!
PR1202

**Comment by Shubhu0** [comment]
Tested the PR, did not notice any errors.

**Review by zuhang2000 — DISMISSED** [review]
Hi I've tested your PR and everything works as expected!

1. The header on dashboard all good. The "missing name" error that they previously mentioned is gone.
Screen Shot 2023-09-17 at 1 42 15 AM

2. No error in the files from header component. Good job!
Screen Shot 2023-09-17 at 1 42 26 AM
Screen Shot 2023-09-17 at 1 42 32 AM

**Review by Shiwani99 — APPROVED** [review]
Hi, 
I tested your PR. It works as intended. There are no errors in files under Header. I also checked if Header folder path was removed from eslintignore and prettierignore. 

Screenshot 2023-09-19 at 8 15 42 PM

Screenshot 2023-09-19 at 8 16 20 PM

**Review by ramyaram2092 — CHANGES_REQUESTED** [review]
Looks good  Shihao. However I still see some warnings like unusued import, unused variable names
image

Screenshot (9)


Screenshot (10)


Screenshot (11)

File: .gitattributes [hunk 1]
```diff
@@ -1 +1 @@
-* text eol=lf
+# * text eol=lf
```
Comment (EvianTan) [inline .gitattributes:1]: See the same comment I left on your #1203 PR.

File: src/components/Reports/PeopleTableDetails.css [hunk 2]
```diff
@@ -1,119 +1,119 @@
-.wrapper.container-fluid {
```
Comment (EvianTan) [inline src/components/Reports/PeopleTableDetails.css:1]: can you help me understand what are the exactly changes in this `.css` file?

**Review by EvianTan — CHANGES_REQUESTED** [review]

**Comment by Shubhu0** [comment]
Hey @sxiong5 I tested your PR, I don't see any errors and everything is displayed as expected. The name appears are it should and the PR is working correctly with no errors.

image

**Comment by palakgosalia** [comment]
Hey, I have checked for errors in the header folder and didn't find any. The App runs as expected. Thank you

image

**Review by savio-henrique — DISMISSED** [review]
Hi @sxiong5 , I've tested your PR, the Header seems to be working as expected, no console errors, and the name seems to be back at the Header after 'Welcome,'. Also no errors on my IDE on Linting the files as shown by the screenshots below.

# Screenshots

working!

No problems

**Review by EvianTan — APPROVED** [review]

**Review by EvianTan — APPROVED** [review]

**Pull request merged**